File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    this.findAll=function () {
        return $http.get('../brand/findAll.do');
    }

    this.findPage=function(pages, rowss){
        return  $http.get('../brand/findPage.do?page=' + pages + '&size=' + rowss);
    }

    this.findOne=function(id){
        return  $http.get('../brand/findOne.do?id='+id);
    }
    this.delete=function (ids) {
        return $http.get('../brand/delete.do?ids='+ids);
    }

    this.add =function (entity) {
        return $http.post('../brand/add.do',entity);
    }
    this.update =function (entity) {
        return $http.post('../brand/update.do',entity);
    }

    this.search=function(pages,rows,searchBrand){
        return  $http.post('../brand/search.do?page=' + pages + '&size=' + rows,searchBrand);
    }

    this.selectBrandList=function () {

        return $http.get('../brand/selectBrandList.do');
    }
});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand> findAll();

    public PageResult findPage(int pageNum,int pageSize);

    public void save(TbBrand tbBrand);

    public void update(TbBrand tbBrand);
    public TbBrand findOne(Long id);

    public void delete(Long[] ids);

    public PageResult findPage(TbBrand tbBrand,int pageNum,int pageSize);

    public List<Map> selectBrandList();

}
